### ISSUE TITLE
Add alt text to all images

### ISSUE THREAD

**Issue by daisypjm** [pat-shannon/mean-money#96]
### Feature

Nav bar

### Suggested Priority

Low

### Observed Behaviour

Currently just the logo! But would benefit from having some alt text added, so that users accessing the site with accessibility tools such as screen readers can easily navigate.

### Expected Behaviour

MeanMoney logo has some alt text added, something simple like "MeanMoney Shark Logo" should be fine.

### Reproduction Steps

1. Check the nav bar from any page
2. Observe the logo/associated code
